runtime/layers/console/bootstrap

```diff
@@ -24,11 +25,16 @@ if (! is_file($handlerFile)) {
 }
 
 while (true) {
-    $lambdaRuntime->processNextEvent(function ($event) use ($handlerFile): array {
+    $lambdaRuntime->processNextEvent(function ($event, Context $context) use ($handlerFile): array {
         $cliOptions = $event['cli'] ?? '';
 
         // We redirect stderr to stdout so that all the output is collected
-        $command = "/opt/bin/php $handlerFile " . $cliOptions . ' 2>&1';
+        $command = sprintf(
+            'LAMBDA_INVOCATION_CONTEXT=%s /opt/bin/php %s %s 2>&1',
```

Comment:
The only fear here is if the environment variable value is very large, but as the JSON encoded context value length is somewhat predictable we should be safe.

An example value looks like:
`{"awsRequestId":"46383f07-e8f9-4f71-b9e1-61279b075bf7","deadlineMs":1587162325806,"invokedFunctionArn":"arn:aws:lambda:us-east-1:262xxxxxx831:function:app-dev-api","traceId":"Root=1-5e9a2cb9-3cf013386c1f89109c927fd0;Parent=3a7eaeb164e48444;Sampled=0"}`
With a letter count of 251